### PR TITLE
update readme for changes

### PR DESCRIPTION
**Pull request by remstone7**
Updates readme w/ how to make a PR and semantic versioning.  Adds changelog

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+### [Unreleased]
+
+
+### [1.2.3] - 2020-10-09
```
Comment (smoucka) [inline CHANGELOG.md:10]: You'll wanna add a link to show the diff between this tag and the previous one

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+##### Changed
+- Removes unused variable and DI from Reclaim.php
+- CSP now uses report-only mode
+
+##### Fixed
+
+
+  
+#### NOTE
+- The CHANGELOG was creeated on 2020-11-20 and does not contain information about earlier releases
```
Comment (smoucka) [inline CHANGELOG.md:22]:
```suggestion
- The CHANGELOG was created on 2020-11-20 and does not contain information about earlier releases
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+### [Unreleased]
```
Comment (smoucka) [inline CHANGELOG.md:7]: Can add a link to the diff between most recently tagged version and HEAD

**Review by smoucka — APPROVED** [review]
Lgtm with the addition of those links

**Review by smoucka — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -54,6 +54,28 @@ Magento 2
     - Open Setup Klaviyo User from the sidebar
     - Enter the username, password, and email for the credentials
     - Save config
+
+
+## Making updates
+1) Follow Klaviyo's standard process of making a pull request and getting it reviewed before merging.
+2) Update CHANGELOG.md. Details on formatting the changelog (including categorizing changes) can be found here: [keepachangelog.com](https://keepachangelog.com/en/1.0.0/)
+    1) If this is a change that will not immediately get sent along to Magento i.e. not a version update:
```
Comment (klaviyojad) [inline README.md:62]: what do you mean not get sent along to magento?  u mean packagist?

File: README.md [hunk 5]
```diff
@@ -54,6 +54,28 @@ Magento 2
...
+        1) Add any changes under the [`[Unreleased]`](https://github.com/klaviyo/magento2-klaviyo/compare/1.0.1...HEAD) section. This will be a comparision of the most recent commits to the latest tagged version.
```
Comment (klaviyojad) [inline README.md:63]: i do not get this whole notion of unreleased, why there going to be PRs approved and not released?

File: README.md [hunk 6]
```diff
@@ -54,6 +54,28 @@ Magento 2
...
+    2) If this is a version update:
+        1) Make sure to increment the version in two places:
+            1) module.xml
+            2) composer.json
+        2) Add a new version between `[Unreleased]` and the most recent version. Include the incremented version number following [semantic versioning](https://semver.org/spec/v2.0.0.html) practices and the date. Add your changes under this version.
+        3) Move any unreleased changes into your version update under the appropriate categories.
+        4) Update the `[Unreleased]` link to point from your new version to HEAD e.g. if you're updating to version 1.0.2 you'd update the link from `1.0.1...HEAD` to `1.0.2...HEAD`.
```
Comment (klaviyojad) [inline README.md:70]: where is this `[Unreleased]` link  ?

Comment (remstone7) [reply README.md:70]: its in the changelog

File: README.md (hunk 5 above)
Comment (remstone7) [reply README.md:63]: this is so you don't need to do a release for every update, Ex. you want to add a couple changes but are separate pr's prior to a release

File: README.md (hunk 4 above)
Comment (remstone7) [reply README.md:62]: Yea Magento marketplace and Packagist

**Pull request merged**